akka-remote/src/main/scala/akka/remote/artery/ArteryTransport.scala

```diff
@@ -284,7 +284,7 @@ private[remote] abstract class ArteryTransport(_system: ExtendedActorSystem, _pr
 
   override val log: MarkerLoggingAdapter = Logging.withMarker(system, classOf[ArteryTransport])
 
-  val flightRecorder: RemotingFlightRecorder = RemotingFlightRecorder.recorder
+  private val flightRecorder = RemotingFlightRecorder
```

Comment:
drop field, and also the debug log statement below, it's there to know which recorder was used but now there is only one